app/src/main/java/org/simple/clinic/util/UuidTypeConverters.kt

```diff
@@ -18,6 +18,32 @@ class UuidRoomTypeConverter {
   }
 }
 
+class UuidListRoomTypeConverter {
```

Comment:
TODO: Discuss about One-To-Many mapping tables instead of storing in a single field.

Reply:
Done